package.json

```diff
@@ -1,27 +1,43 @@
 {
   "name": "@nebulr-group/nblocks-ts-client",
-  "version": "5.4.1",
-  "description": "This library contains the low level NBlocks client that is shipped with all of our NBlocks plug-n-play experiences for different technologies.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
+  "version": "5.4.2",
+  "exports": {
+    "./engine": {
+      "types": "./dist/engine/index.d.ts",
+      "require": "./dist/engine/index.js",
+      "default": "./dist/engine/index.js"
+    },
+    "./core-api": {
+      "types": "./dist/core-api/index.d.ts",
+      "require": "./dist/core-api/index.js",
+      "default": "./dist/core-api/index.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "engine": ["./dist/engine/index.d.ts"],
+      "core-api": ["./dist/core-api/index.d.ts"]
+    }
+  },  
+  "description": "This library contains the low level NBlocks client that is shipped with all of our NBlocks plug-n-play experiences for different technologies.",  "files": [
     "/dist"
   ],
   "contributors": [
-    "Oscar Söderlund <oscar@nebulr.group>"
+    "Oscar Söderlund <oscar@nebulr.group>",
+    "Iman Pouya <iman@nebulr.group>"
   ],
   "directories": {
     "example": "examples"
   },
   "scripts": {
-    "prebuild": "rimraf dist",
+    "prebuild": "rm -rf dist/*",
     "build": "tsc",
     "npm-publish": "npm run build && npm publish --access public",
     "npm-publish-beta": "npm run build && npm publish --access public --tag beta",
-    "npm-publish-local": "npm run build && npm publish --registry http://verdaccio:4873 --tag alpha",
+    "npm-publish-local": "npm run build && npm publish --registry  http://host.docker.internal:4873  --tag alpha",
```

Comment:
I'd like some insights here and how this works

Reply:
using host.docker.internal containers can talk without being on the same docker network. Really handy especially for verdaccio. No need to connect this tsc project and the verdaccion to the same docker network